.github/workflows/book.yml

```diff
@@ -23,6 +23,7 @@ jobs:
 
       - name: Setup mdBook
         run: |
+          # TODO: use nix
```

Comment:
Same, is the PR planned already?

Reply:
I did not planned. We did not have planning. I am not sure what to do with it? We have 3+soon 2 SRE people. Would they help? I mean I will fix what is must for XCVM and XCMP, so is this nix stuff in scope of that area? 
we can discuss scoping on CI NIX review meeting.